src/cliOptions.ts

```diff
@@ -41,6 +41,12 @@ export const optionDefinitions = [
     type: Boolean,
     description: 'Add missing keys that were found by the detective (default value is false)'
   },
+  {
+    name: 'delete-missing-keys',
+    alias: 'D',
```

Comment:
The alias might be a bit confusing no? What do you think?

Reply:
Went with `-D` because `-d` is already used. We could also go with `-A` (kind of opposite of the `-a` for "Add missing keys" above. Or none, at all :)